crates/ide_assists/src/handlers/generate_getter.rs

```diff
@@ -136,7 +136,7 @@ pub(crate) fn generate_getter_impl(
     )
 }
 
-fn useless_type_special_case(field_name: &str, field_ty: &ast::Type) -> Option<(String, String)> {
+pub fn useless_type_special_case(field_name: &str, field_ty: &String) -> Option<(String, String)> {
```

Comment:
Let's pull this up into `assists/src/utils.rs` -- there shouldn't be uses "across" assists.